modules/twinklewarn.js

```diff
@@ -297,63 +367,63 @@ Twinkle.warn.messages = {
 				label: "Page moves against naming conventions or consensus",
 				summary: "General note: Page moves against naming conventions or consensus"
 			},
-			"uw-tpv1": {
-				label: "Refactoring others' talk page comments",
-				summary: "General note: Refactoring others' talk page comments"
-			},
-			"uw-upload1": {
-				label: "Uploading unencyclopedic images",
-				summary: "General note: Uploading unencyclopedic images"
-			}
-		}/*,
-		"To be removed from Twinkle": {
 			"uw-redirect1": {
 				label: "Creating malicious redirects",
 				summary: "General note: Creating malicious redirects"
 			},
```

Comment:
I agree with the original code that this probably doesn't belong in Twinkle. How often does this happen? Wouldn't "uw-vandalism1" or the like work just as well, if not the speedy deletion notice?